solr/solr-ref-guide/src/jdbc-dbvisualizer.adoc

```diff
@@ -22,8 +22,8 @@ For DbVisualizer, you will need to create a new driver for Solr using the DbVisu
 This will add several SolrJ client .jars to the DbVisualizer classpath.
 The files required are:
 
-* all .jars found in `$SOLR_HOME/dist/solrj-lib`
-* the SolrJ .jar found at `$SOLR_HOME/dist/solr-solrj-<version>.jar`
+* all .jars found in `$SOLR_HOME/server/solr-webapp/webapp/WEB-INF/lib/\*` and `$SOLR_HOME/server/lib/ext/*`
```

Comment:
`lib/ext/*` is for the two slf4j jars that are included there, but not in `WEB-INF/lib`. I don't know, I just added it, since there are solrj dependencies in each. It was probably a mistake in documentation?

Good call on `SOLR_TIP`, I'll fix that.

Reply:
Seeing above line 22 "all JARs" on these paths... well... like Jan commented above, it isn't actually all of them.  Nonetheless I could see that for a JdbcVisualizer how-to, that practically speaking it may be easiest to add them all? (did you try this?).  It'd be nice if we could have gradle dump a text file of the dependency list so the user knows which are needed.

Reply:
Ok I included the paragraph talking about how to find the necessary dependencies from below. (And included it in all locations where these bullet points are found).

Reply:
I'm still not convinced about `server/lib/ext/*`. They are logging jars and slf4j bindings, while the `slf4j-api` is in solr-core.jar. Does it even make sense for JDBC driver to log anything anywhere? Where would that log end up? If you do NOT include those jars, then the slf4j-api code would simply output to sterr a warning that no logger binding was found for slf4j, so no logging will be done. Which is probably ok for a jdbc driver?

Reply:
Candidate for testing. Or ask the one who wrote the initial Solr-JDBC driver docs?

Reply:
I probably initially wrote these docs (or at least contributed over time) and at the time it was almost all jars in those places. It was easier to say "use all" instead of trying to list specific ones to pull out.

It would be 10000000x better to have a single jdbc jar with everything included - SOLR-8680 was filed years ago to try to address this. I couldn't figure out a good way to do it with ant, but it might be easier now with gradle and the improved dependency management logic (ie: ability to include transitive). I haven't looked recently.

There are probably a ton of jars that aren't needed, but requires testing to figure out the minimal set or potentially using gradle dependency management to simplify.  The ant build made it VERY hard to try to pull out specific dependencies. If gradle can build a dependency tree to see what gets pulled in - it would be a lot easier.

Reply:
+1 to a fat jdbc jar. But I'm reluctant to adding it to the fat tgz which will be even fatter.
Or could be make the build only push the fat-jar to maven, without including it in the tgz? Then the docs could tell people that a fat jdbc jar exists, and we could include its SHA-sum in the tgz to testify to its validity?

This is a good example of a jar that is never needed by Solr server side, so making it a plugin package won't make sense. Maven is the way to go, and, if required, upload the jar as a separate jar or tgz artifact in the release repo.
When we split solrj into even more specialized jars, perhaps we'd also want to provide a single `solrj-all.jar` for convenience.

Reply:
Am I interpreting this correctly that this is a future improvement (hopefully early-on in the 9.x release cycle)? Or is this a blocker for this ticket?

Reply:
Not a blocker.